docs/administrators/on_disk_layout.rst

```diff
@@ -0,0 +1,70 @@
+On Disk Layout
+====================
+
+The server stores
+ - all data below the state directory, as configured by :inmanta.config:option:`config.state-dir`,
+which defaults to `/var/lib/inmanta`
+ - all logs below the log directory, as configured by  :inmanta.config:option:`config.log-dir`,
+which defaults to `/var/log/inmanta`
```

Comment:
This itemization renders weird. It think this is caused by the line break within the content of one item.